devsite/devsite/settings.py

```diff
@@ -41,10 +37,18 @@
     'rest_framework',
     'django_countries',
     'django_filters',
-    #'rest_framework.authtoken',
+    # 'rest_framework.authtoken',
     'webpack_loader',
     'devsite',
     'figures',
+
+    # edx-platform apps. Mocks are used by default
```

Comment:
Nope, don't have docs for testing yet. In the backlog. I can reword this (remove the 'used by default') as only mocks are used in devsite since its purpose is to provide a simple (and transparent to Figures) simulation of edx-platform